dev_tools/locustfile.py

```diff
@@ -0,0 +1,36 @@
+"""Locust (https://docs.locust.io/en/stable/what-is-locust.html) is a load testing tool.
+To run Locust, set the SUPERSTAQ_API_KEY environment variable and execute
+"locust --config=locust.conf" in the "dev_tools" directory. Then
+navigate to http://0.0.0.0:8089/, insert the number of users and spawn rate,
+then click the "Start swarming" button.
+"""
+
+from __future__ import annotations
+
+import logging
+
+import cirq_superstaq
+import locust.env
+import requests
+
+
+@locust.events.quitting.add_listener
+def _(environment: locust.env.Environment) -> None:
+    if environment.stats.total.fail_ratio > 0.01:
+        logging.error("Test failed due to failure ratio > 1%")
+        environment.process_exit_code = 1
+    else:
+        environment.process_exit_code = 0
+
+
+SERVICE = cirq_superstaq.Service()
```

Comment:
```suggestion
SERVICE = css.Service()
```

Reply:
also... `service`? Again for consistency with our examples. Also, I don't think of service as a constant (which would be all caps)

Reply:
All caps cause it's a global var in this file.